README.rst

```diff
@@ -0,0 +1,54 @@
+python3-pip-skeleton
+===========================
+
+|code_ci| |docs_ci| |coverage| |pypi_version| |license|
+
+.. note::
+    
+    This project contains template code only. For documentation on how to
+    adopt this skeleton project see 
+    https://epics-containers.github.io/python3-pip-skeleton-cli
+
+    [Remove this note when adopting skeleton.]
```

Comment:
Rather than this last sentence, could add a test to test_boilerplate_removed to check the note is removed

Reply:
done